### DISCUSSION TITLE
ADR Suggestion, Convert ADRs to issues - extension of ADR #15

### DISCUSSION THREAD

**Discussion by rozyczko** [easyscience/.github#18] [Ideas]
Many ADRs result in issues to be implemented. This can easily be done from the ADR page itself, with the `Create issue from discussion` action.
The proposed extension to #15 is:

- Once the ADR discussion is agreed and the ADR is created, a relevant issue should be created for tracking using the standard project dashboard.

This will help in having the project properly scoped and with having the full list of changes for the following release.

**Comment by damskii9992** [comment]
@AndrewSazonov If you have no issue with this ADR suggestion, I'd like to close this discussion and ammend the corresponding ADR.

**Comment by AndrewSazonov** [comment]
Okay, I’ll add the following new section to the existing ADR, unless there are any objections:

## Tracking Implementation

1. Once the ADR is accepted and finalized, create a corresponding issue in the relevant repository (or repositories) to track its implementation. This can be done using the `Create issue from discussion` button on the right sidebar of the ADR discussion page.
2. If the ADR impacts multiple projects or components, create an **epic** or **umbrella issue** that includes a checklist of all sub-issues or tasks needed to implement the changes across affected codebases and documentation.
